### ISSUE TITLE
Event functions with parameters

### ISSUE THREAD

**Issue by davidbrooksio** [cferdinandi/reef#186]
Hi, this is more of a question (sorry) and therefore I haven't created a reduced test case but happy to create one if deemed necessary.

Firstly, thank you for fixing issue #184, this kind of relates to that fix.

Simply put; is it possible to have an onclick event that has parameters? At the moment it seems not as the only parameter is the event itself, I assume this is for XSS prevention but please can you confirm that this is the case.

An example of parameters is:

```
<button onclick="myFunction('Hello world')">Click me</button>

------

function myfunction(greeting) {
    console.log(greeting);
}
```

Thank you

**Comment by cferdinandi** [comment]
I think that could be supported! Flagging this is a feature request/bug.

**Comment by cferdinandi** [comment]
Addressed in version 12.5.0
